api/src/main/java/org/openmrs/BaseOpenmrsMetadata.java

```diff
@@ -40,8 +41,9 @@ public abstract class BaseOpenmrsMetadata extends BaseOpenmrsObject implements O
 	@FullTextField(analyzer = SearchAnalysis.NAME_ANALYZER)
 	private String name;
 	
-	@Column(name = "description", length = 255)
 	@Lob
+	@Type(type = "large_text")
```

Comment:
@Type is deprecated in Hibernate 6. Is @Column(columnDefinition="text") an option?

Reply:
We're still on Hibernate 5.6.15.Final, so `@Type(type = "large_text")` is valid. Shouldn't we handle Hibernate 6 when we actually upgrade or in another commit? I am happy to update it if you think it's feasible.

Reply:
columnDefinition is available in 5.6 so please use it.